platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -532,13 +589,28 @@ namespace platoon_strategic_ihp
             * \return mobility operation msg.
             */
             cav_msgs::MobilityOperation composeMobilityOperationLeaderAborting();
-            
             /**
             * \brief Function to compose mobility operation in CandidateLeader.
             *
             * \return mobility operation msg.
             */
             cav_msgs::MobilityOperation composeMobilityOperationCandidateLeader();
+            /**
+            * \brief Function to compose mobility operation in LeadWithOperation (cut-in join)
```

Comment:
In these two methods, the input string has a risk of being malformed.  How will the methods handle garbage input?  If it throws and exception or returns null, etc, this condition needs to be documented here.

Reply:
Updated.